uaclient/cli.py

```diff
@@ -487,6 +487,13 @@ def get_parser():
         action="store_true",
         help="show all debug log messages to console",
     )
+    parser.add_argument(
+        "-v",
```

Comment:
let's just do the --version, and skip the -v as that can be conflated with verbosity levels.